src/components/misc/category/Header.tsx

```diff
@@ -24,10 +30,12 @@ const Header = ({
       setOpened('')
     } else {
       setOpened(section)
-      ref.current?.scrollIntoView({
-        block: 'start',
-        behavior: 'smooth',
-      })
+      if (ref.current && ref.current.scrollIntoView) {
```

Comment:
ok, j'aurai besoin d'une explication pour le comportement du ref ici :)

Reply:
https://react.dev/learn/manipulating-the-dom-with-refs#example-scrolling-to-an-element